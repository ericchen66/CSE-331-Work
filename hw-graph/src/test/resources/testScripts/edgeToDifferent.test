# Tests a parent node that has edges to two
# other nodes. Ensures parent/child relation holds
# for all nodes

# Creates an empty graph
CreateGraph graph1

# Adds three nodes to the graph
AddNode graph1 node1
AddNode graph1 node2
AddNode graph1 node3

# Creates two edges, both originating from the same node
# but connecting to two different nodes
AddEdge graph1 node1 node2 edge1
AddEdge graph1 node1 node3 edge2

# Lists the children of the parent node
ListChildren graph1 node1