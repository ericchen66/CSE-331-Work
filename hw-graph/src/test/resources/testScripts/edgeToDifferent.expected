# Tests a parent node that has edges to two
# other nodes. Ensures parent/child relation holds
# for all nodes

# Creates an empty graph
created graph graph1

# Adds three nodes to the graph
added node node1 to graph1
added node node2 to graph1
added node node3 to graph1

# Creates two edges, both originating from the same node
# but connecting to two different nodes
added edge edge1 from node1 to node2 in graph1
added edge edge2 from node1 to node3 in graph1

# Lists the children of the parent node
the children of node1 in graph1 are: node2(edge1) node3(edge2)